test/test-stripe.py

```diff
@@ -0,0 +1,12 @@
+import stripe
+stripe.api_key = "sk_test_BQokikJOvBiI2HlWgH4olfQ2"
```

Comment:
uh... regenerate this key and take an environment variable for this. even if it's a test key we shouldn't commit it to version control  :)